### PR TITLE
(fix #29939) naming of startpage targets in personal settings

### PR DESCRIPTION
**Pull request by utesche**
Fixes naming of startpage targets in personal settings,
see mantis: https://mantis.ilias.de/view.php?id=29939

### CODE REVIEW

File: Services/User/classes/class.ilUserUtil.php [hunk 1]
```diff
@@ -227,7 +227,7 @@ public static function getPossibleStartingPoints($a_force_all = false)
         
         $all = array();
         
-        $all[self::START_PD_OVERVIEW] = 'overview';
+        $all[self::START_PD_OVERVIEW] = 'obj_dshs';
```
Comment (smeyer-ilias) [inline Services/User/classes/class.ilUserUtil.php:230]: "obj_dshs" is the translation of the "Dashboard Administration Node". 
I suggest to use the language variable "dash_dashboard" instead from language module 'dash'

Please also add $lng->loadLanguageModule('dash');

File: Services/User/classes/class.ilUserUtil.php [hunk 2]
```diff
@@ -248,7 +248,7 @@ public static function getPossibleStartingPoints($a_force_all = false)
             $all[self::START_PD_CALENDAR] = 'calendar';
         }
 
-        $all[self::START_REPOSITORY] = 'repository';
+        $all[self::START_REPOSITORY] = 'obj_root';
```
Comment (smeyer-ilias) [inline Services/User/classes/class.ilUserUtil.php:251]: Yes, that's better

**Review by smeyer-ilias — CHANGES_REQUESTED** [review]
Thanks for fixing this and creating a PR!

File: Services/User/classes/class.ilUserUtil.php (hunk 1 above)
Comment (utesche) [reply Services/User/classes/class.ilUserUtil.php:230]: ok,  I orientated my research to use a 'common' langvar, if possible.

**Comment by alex40724** [comment]
Thanks for the PR. Fixed in git.

**Pull request closed without merging**